include/deal.II/base/mpi.h

```diff
@@ -357,24 +357,45 @@ namespace Utilities
     /**
      * A class that is used to initialize the MPI system at the beginning of a
      * program and to shut it down again at the end. It also allows you to
-     * control the number threads used in each MPI task.
+     * control the number of threads used within each MPI process.
      *
-     * If deal.II is configured with PETSc, the library will be
-     * initialized in the beginning and destroyed at the end automatically
-     * (internally by calling PetscInitialize() and PetscFinalize()).
+     * If deal.II is configured with PETSc, PETSc will be initialized
+     * via `PetscInitialize` in the beginning (constructor of this
+     * class) and de-initialized via `PetscFinalize` at the end (i.e.,
+     * in the destructor of this class). The same is true for SLEPc.
      *
      * If deal.II is configured with p4est, that library will also be
-     * initialized in the beginning, and destroyed at the end automatically
-     * (internally by calling sc_init(), p4est_init(), and sc_finalize()).
-     *
-     * If a program uses MPI one would typically just create an object of this
-     * type at the beginning of <code>main()</code>. The constructor of this
-     * class then runs <code>MPI_Init()</code> with the given arguments. At
-     * the end of the program, the compiler will invoke the destructor of this
-     * object which in turns calls <code>MPI_Finalize</code> to shut down the
-     * MPI system.
-     *
-     * This class is used in step-32, for example.
+     * initialized in the beginning, and de-initialized at the end
+     * (by calling sc_init(), p4est_init(), and sc_finalize()).
+     *
+     * If a program uses MPI one would typically just create an object
+     * of this type at the beginning of <code>main()</code>. The
+     * constructor of this class then runs <code>MPI_Init()</code>
+     * with the given arguments and also initializes the other
+     * libraries mentioned above. At the end of the program, the
+     * compiler will invoke the destructor of this object which in
+     * turns calls <code>MPI_Finalize</code> to shut down the MPI
+     * system.
+     *
+     * This class is used in step-17, step-18, step-40, step-32, and
+     * several others.
+     *
+     * @note This class performs initialization of the MPI subsystem
+     * as well as the dependent libraries listed above through the
+     * `MPI_COMM_WORLD` communicator. This means that you will have to
```

Comment:
Do our doxygen scripts understand `? 

I would write `<code>MPI_COMM_WORLD</code>` to be consistent with the rest of the file.

Reply:
Yes, and we've started using that in a number of places already.